docs_src/pipelinebenchmarking.md

```diff
@@ -129,6 +130,28 @@ sudo ./benchmark.sh --stream_density <target FPS> --logdir <output dir>/data --i
 **_NOTE:_** Because stream density requires a continuous video stream it is recommended to use an RTSP stream, USB camera, or RealSense camera. If these options are not available you can use the [camera simulator](#appendix-benchmark-helper-scripts) to continuously loop through a video file as an RTSP stream.
 
 ---
+
+#### Workload
+We are currently supporting 2 types of workload:
+    1. dlstreamer
+    2. opencv-ovms
```

Comment:
why is it opencv-ovms ? should it be openvino-ovms ? I am assuming either pipelines would be using opencv for image processing.

Reply:
that's how we named the opencv-ovms workload, @jim-wang-intel do you know why we name it this way?

Reply:
that is original name came up by Jubilee, but it is not accurate to be openvino-ovms strictly speaking as it uses opencv in some of profiles as well

Reply:
Both dlstreamer and ovms use openCV and openVINO for inferencing.
The main different is the abstraction layer of how openVINO engine is used.
I would just call it: OVMS or spell it out: openVINO Model Server

Reply:
I like Antonio's suggestion, just keep it as OVMS or OpenVINO Model Server.